python/lsst/pipe/tasks/multiBand.py

```diff
@@ -539,7 +539,7 @@ def __init__(self, butler=None, schema=None, peakSchema=None, refObjLoader=None,
             schema = initInputs['inputSchema'].schema
         if schema is None:
             assert butler is not None, "Neither butler nor schema is defined"
-            schema = butler.get(self.config.coaddName + self.inputCatalog + "_schema", immediate=True).schema
+            schema = butler.get(self.config.coaddName + self.inputCatalog + "_schema").schema
```

Comment:
Wait, are tasks ever initialized with a butler in gen3?

Reply:
I was wondering. The code is clearly never called because if it was called it would break... I wasn't brave enough to remove the butler parameter completely from the 8 places that use it but maybe I should.

Reply:
I don't think I want to get into that on this ticket though.